superbench/benchmarks/micro_benchmarks/gpu_copy_performance/gpu_copy.cu

```diff
@@ -226,7 +232,7 @@ int SetGpu(int gpu_id) {
 }
 
 // Prepare data buffers and streams to be used.
-int PrepareBufAndStream(BenchArgs *args) {
+int PrepareResources(BenchArgs *args) {
```

Comment:
I think you need to have a new function to PrepareEvent() and DestroyEvent() without modifying original code.

Reply:
Fixed